### DISCUSSION TITLE
default / medium_quality works better than 'best_quality'?

### DISCUSSION THREAD

**Discussion by johncantrell97** [autogluon/autogluon#2043] [Q&A]
Hi,

I trained a tabular classification model using the default settings and get an accuracy higher than when I train it using presets = 'best_quality'.  It's about 4% higher using the default (medium_quality?).  

Is there an explanation for this? I tried giving it more time (30 minutes) but it didn't use all of it and still performed worse when set to best quality.

Any ideas?

**Comment by Innixma** [comment]
Please provide full logs of training + predictor.leaderboard(test_data) output for both medium and best quality runs so I can investigate the reason. If possible also provide a reproducible script via colab that includes the data.
